Google.PowerShell/PubSub/GcpsSubscription.cs

```diff
@@ -315,4 +314,139 @@ private void GetSubscriptions(IEnumerable<string> subscriptionNames)
             }
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Changes the config of a subscription.
+    /// </para>
+    /// <para type="description">
+    /// Changes the config of a subscription from push to pull and vice versa. The cmdlet can also be used to
+    /// change the endpoint of a push subscription. Will raise error if the subscription cannot be found.
+    /// </para>
+    /// <example>
+    ///   <code>PS C:\> Set-GcpsSubscriptionConfig -Subscription "my-subscription" -PullConfig</code>
+    ///   <para> This command sets the config of subscription "my-subscription" in the default project to pull config.</para>
+    /// </example>
+    /// <example>
+    ///   <code>PS C:\> Set-GcpsSubscriptionConfig -InputObject $subscription -PullConfig</code>
```

Comment:
The -InputObject parameter is mostly used for pipelining from Get-GcpsSubscription. I removed this example since I think this is not a typical use case (the typical use case will be the pipelining one).